rotkehlchen/chain/ethereum/modules/diva/decoder.py

```diff
@@ -0,0 +1,136 @@
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+        )
+        self.diva = A_DIVA.resolve_to_evm_token()
+
+    def _process_delegation_change(
+            self,
+            topics: list[bytes],
+            transaction: EvmTransaction,
+            tx_log: EvmTxReceiptLog,
+    ) -> DecodingOutput:
+        """Process information for changes in delegation"""
+        delegator = hex_or_bytes_to_address(topics[1])
+        delegate = hex_or_bytes_to_address(topics[3])
+
+        if self.base.is_tracked(delegator):
+            event_address = delegator
+        else:
+            event_address = delegate
+
+        event = self.base.make_event_from_transaction(
+            transaction=transaction,
+            tx_log=tx_log,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.GOVERNANCE,
+            asset=self.diva,
+            balance=Balance(),
+            location_label=event_address,
+            notes=f'{delegator} started to delegate on {self.diva.symbol} to {delegate}',
+            counterparty=CPT_DIVA,
+        )
+        return DecodingOutput(event=event, refresh_balances=False)
+
+    def _decode_delegation_change(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a change in the delegated address"""
+        if context.tx_log.topics[0] != DELEGATE_CHANGED:
+            return DEFAULT_DECODING_OUTPUT
+
+        return self._process_delegation_change(
+            topics=context.tx_log.topics,
+            transaction=context.transaction,
+            tx_log=context.tx_log,
+        )
+
+    def _decode_diva_claim(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] == DELEGATE_CHANGED:
+            return self._process_delegation_change(
+                topics=context.tx_log.topics,
+                transaction=context.transaction,
+                tx_log=context.tx_log,
+            )
+        elif context.tx_log.topics[0] != CLAIM_AIRDROP:
+            return DEFAULT_DECODING_OUTPUT
+
+        claiming_address = hex_or_bytes_to_address(context.tx_log.data[32:64])
+        claimed_amount = token_normalized_value(
+            token_amount=hex_or_bytes_to_int(context.tx_log.data[64:]),
+            token=self.diva,
+        )
+
+        for event in context.decoded_events:
+            if (
+                event.event_type == HistoryEventType.RECEIVE and
+                event.location_label == claiming_address and
+                event.asset == self.diva and
+                event.balance.amount == claimed_amount
+            ):
+                event.event_type = HistoryEventType.RECEIVE
+                event.event_subtype = HistoryEventSubType.AIRDROP
+                event.counterparty = CPT_DIVA
+                event.notes = f'Claim airdrop of {event.balance.amount} {self.diva.symbol}'
```

Comment:
Same as before for consistency. In optimism it's

`Claimed {amount} OP from optimism airdrop`

But perhas we should indeed change to present?

```suggestion
                event.notes = f'Claim {event.balance.amount} DIVA from the DIVA airdrop'
```